resource/compute_dynamic_select.py

```diff
@@ -0,0 +1,24 @@
+# display options depending on model provider availability
+model_provider = None
+
+try:
+    from dataiku.core.model_provider import get_model_provider
+    model_provider = get_model_provider()
+except ImportError:
+    pass
```

Comment:
@AlxBit do you think it's a reliable way to maintain backward compatibility with dss version 12.0.x which do not contain the `model_provider`?

Reply:
> Look Before You Leap

vs 

> Easier to Ask Forgiveness Than Permission

🍿

Reply:
@AlxBit for which versions of dss do you guarantee availability of the model cache?

Reply:
I have a slight preference for the current check because:
- It requires less assumptions about what is available in which DSS version
- DSS version is not always easy to compare (e.g. `0.0.0-ci-dip-on-demand-13311` for a dev build -> how would you handle it ?)

Reply:
The model cache is available since 12.1 but I think even if we were pick the version checking we'd still want the current check as Fred pointed out.

Reply:
all good then